apps/nowcasting-app/components/charts/forecast-header/index.tsx

```diff
@@ -22,14 +22,10 @@ const ForecastHeader: React.FC<ForecastHeaderProps> = ({
   const timeNow = useTimeNow();
 
```

Comment:
hey @peterdudfield, everything looks good to me. just from a clean up perspective, I don't think `useMemo` is being used. And on the `ForecastHeaderProps` type, `pvUpdatedData` and `selectedTime` end up not being used.

Reply:
Thanks, Ill remove 'useMemo', and remove those other varibales. 

I'm interested in how you knew they weren't being used? Was that something that came up on VS code?